src/main/java/seedu/address/logic/Logic.java

```diff
@@ -16,10 +16,11 @@
 public interface Logic {
     /**
      * Executes the command and returns the result.
+     *
      * @param commandText The command as entered by the user.
      * @return the result of the command execution.
      * @throws CommandException If an error occurs during command execution.
-     * @throws ParseException If an error occurs during parsing.
+     * @throws ParseException   If an error occurs during parsing.
```

Comment:
There is a space here, not sure if it is intended?

Reply:
it is for checkstyle i think